source/module_hamilt_lcao/module_hcontainer/atom_pair.h

```diff
@@ -8,6 +8,7 @@
 #include <vector>
 #include <complex>
 #include <tuple>
+#include <cassert>
```

Comment:
It looks no need to add this line.

Reply:
Compilation error:
https://github.com/deepmodeling/abacus-develop/blob/f0ce6432a82cafd33c67c1e51f3d687f3b883789/source/module_hamilt_lcao/module_hcontainer/atom_pair.h#L265